Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md

```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../Set-Up/Power-Tools/Generating-UDA-files/#what-are-uda-files) article.
 
-* Make sure the folder `/data/revision` on your Umbraco Cloud project is empty
+* Make sure the folder `/umbraco/Deploy/revision` on your Umbraco Cloud project is empty
   * If you have any files in the folder, you can safely remove those at this point
-* Open a command prompt
-* Navigate to the `/data` folder in your local Umbraco Cloud project files
-* Add an *export* marker by typing `echo > deploy-export`
+* Start your local cloud project
+* Navigate to the settings tab in your project
+* Go to the Deploy tab in the settings section and run the `Extract Schema To Data Files` Command
```

Comment:
Suggestion: Split the sentence and bold the section/tab drop-down names

Go to the **Deploy** tab in the **Settings** section. Select `Extract Schema To Data Files` from the **Deploy Operations** drop-down and click **Trigger Operation**.

Reply:
Makes sense, thanks!